setup.md

```diff
@@ -4,93 +4,82 @@ This document will guide you through setting up Fennel on your
+4. Run `gpg --verify fennel-0.4.1.asc` to verify that the fennel script is from the Fennel creators
+5. Move `fennel-0.4.1` to a directory on your `$PATH`, such as `/usr/local/bin`
 
-1. `cd` to a directory in which you want to download Fennel, such as
-   `~/src`
-2. Run `git clone https://github.com/bakpakin/Fennel`
+**Note**: You can rename the script to `fennel` for convenience.
 
-### To install Fennel
+## Downloading a Fennel binary
 
-1. Run `cd Fennel`
-2. Run `make fennel`
-3. Copy or link the `fennel` script to a directory on your `$PATH`,
-   such as `/usr/local/bin`
+Downloading a Fennel binary allows you to run Fennel on your computer without
+having to download Lua.
 
-**Note 1**: Step 2 above will compile Fennel into a standalone script
-called `fennel`.
+### To download a Fennel binary
 
-**Note 2**: If the `fennel` script exists in one of the directories on
-your `$PATH` , you can run `fennel filename.fnl` to run a Fennel file.
+1. Choose one the options below, depending on your system:
+   - [GNU/Linux x86_64](https://fennel-lang.org/downloads/fennel-0.4.1-x86_64) ([signature](https://fennel-lang.org/downloads/fennel-0.4.1-x86_64.asc))
+   - [GNU/Linux arm32](https://fennel-lang.org/downloads/fennel-0.4.1-arm32) ([signature](https://fennel-lang.org/downloads/fennel-0.4.1-arm32.asc))
+   - [Windows x86 32-bit](https://fennel-lang.org/downloads/fennel-0.4.1-windows32.exe) ([signature](https://fennel-lang.org/downloads/fennel-0.4.1-windows32.exe.asc))
 
-## Using LuaRocks to download and install Fennel
+## Downloading Fennel using LuaRocks
 
 [LuaRocks](https://luarocks.org/) contains a repository of Lua
 software packages. LuaRocks is convenient because it automates the
 downloading, installation, and uninstallation of Lua software packages.
 
-### To download and install Fennel
+### To download Fennel using LuaRocks
 
 1. Ensure the `luarocks` package is installed on your system
 2. Ensure the `~/.luarocks/bin` directory is added to your shell's `$PATH`
-3. Run one of
+3. Run one of the following commands:
     - `luarocks install fennel` to install globally (requires admin or sudo)
-    - `luarocks --local install fennel` to install locally (see the note below about paths)
+    - `luarocks --local install fennel` to install locally
+4. Run `fennel --help` to confirm the installation succeeded
 
-**Note:** When using luarocks with `--local`, you'll need to ensure you have
-the correct lua path values. You can run `eval $(luarocks path --bin)` from
-your shell to set this correctly; see [LuaRocks path docs](https://github.com/luarocks/luarocks/wiki/path)
-for details.
-
-You can try running `fennel --help` to confirm the
-installation succeeded.
+**Note:** When using `luarocks` with `--local`, you'll need to ensure
+you have the correct Lua path values. You can run `eval $(luarocks
+path --bin)` to set this correctly. See the [LuaRocks path docs](https://github.com/luarocks/luarocks/wiki/path)
```

Comment:
Rather than "You can run ..." maybe "You can add ... to your shell's init file" would be better, so it's clearer how to make it stick for future sessions.